packages/shorebird_cli/lib/src/commands/release_new/release_type.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// The different types of shorebird releases that can be created.
+enum ReleaseType {
+  /// An Android archive used in a hybrid app.
+  aar,
+
+  /// A full Flutter Android app.
+  android,
+
+  /// A full Flutter iOS app.
+  ios,
+
+  /// An iOS framework used in a hybrid app.
+  iosFramework;
+
+  /// The CLI argument used to specify the release type(s).
+  String get cliName {
+    switch (this) {
+      case ReleaseType.android:
+        return 'android';
+      case ReleaseType.ios:
+        return 'ios';
+      case ReleaseType.iosFramework:
+        return 'ios-framework';
+      case ReleaseType.aar:
+        return 'aar';
+    }
+  }
+
+  /// The platform associated with the release type.
+  ReleasePlatform get releasePlatform {
+    switch (this) {
+      case ReleaseType.aar:
+        return ReleasePlatform.android;
+      case ReleaseType.android:
+        return ReleasePlatform.android;
+      case ReleaseType.ios:
+        return ReleasePlatform.ios;
+      case ReleaseType.iosFramework:
+        return ReleasePlatform.ios;
+    }
+  }
+
+  /// Whether --release-version must be specified to patch.
```

Comment:
```suggestion
  /// Whether --release-version must be specified to release.
```
I think you meant "to release"?

Reply:
Yep. Deleted this whole thing, as I had unintentionally introduced two copies of this function here and in the releaser.